assets/redirects/redirects.csv

```diff
@@ -276,4 +276,5 @@
 /releases/educationcensus2021,/releases/educationcensus2021inenglandandwales
 /releases/healthcensus2021,/releases/healthdisabilityandunpaidcarecensus2021inenglandandwales
 /releases/initialfindingsfromcensus2021,/releases/initialfindingsfromthe2021censusinenglandandwales
-/census/topicsummaries,/census/aboutcensus/censusproducts/topicsummaries
\ No newline at end of file
+/census/topicsummaries,/census/aboutcensus/censusproducts/topicsummaries
+/veteranssurvey,/surveys/informationforhouseholdsandindividuals/householdandindividualsurveys/veteranssurvey
```

Comment:
It looks like there should be a new line at the end of the file.

Reply:
There was no empty line before. I suppose it could fail if it found an empty one?